.github/workflows/build-image.yml

```diff
@@ -44,12 +44,12 @@ jobs:
           dockerfile: rocm5.7.x
           tags: superbench/main:rocm5.7
           runner: [self-hosted, rocm-build]
-          build_args: "NUM_MAKE_JOBS=64"
-        - name: rocm6.0
-          dockerfile: rocm6.0.x
-          tags: superbench/main:rocm6.0
+          build_args: "NUM_MAKE_JOBS=16"
+        - name: rocm6.2
+          dockerfile: rocm6.2.x
```

Comment:
We only release the latest rocm image?

Reply:
we will release one old(rocm6.0) and one latest(rocm6.2)